webpack.config.js

```diff
@@ -1,100 +0,0 @@
-const path = require('path');
-const StyleLintPlugin = require('stylelint-webpack-plugin');
-const VisualizerPlugin = require('webpack-visualizer-plugin');
-
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 2900000;
-const MAX_PRODUCTION_OUTPUT_SIZE = 335000;
-
-module.exports = (env, argv) => ({
-  devtool: argv.mode === 'production'
-    ? false
-    : 'eval-cheap-module-source-map',
-  entry: {
-    lib: [
-      path.join(__dirname, 'src/lib/index.js'),
-    ],
-  },
-  externals: {
-    react: {
-      amd: 'react',
-      commonjs: 'react',
-      commonjs2: 'react',
-      root: 'React',
-      umd: 'react',
-    },
-    'react-dom': {
-      amd: 'react-dom',
-      commonjs: 'react-dom',
-      commonjs2: 'react-dom',
-      root: 'ReactDOM',
-      umd: 'react-dom',
-    },
-  },
-  module: {
-    rules: [
-      {
-        include: path.join(__dirname, 'src/lib'),
-        test: /\.(js|jsx)$/,
-        use: [{ loader: 'babel-loader' }],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          {
-            loader: 'css-loader',
-            options: {
-              modules: {
-                localIdentName: argv.mode === 'production'
-                  ? '[hash:base64:8]'
-                  : '[name]__[local]__[hash:base64:8]',
-              },
-            },
-          },
-          { loader: 'postcss-loader' },
-          {
-            loader: 'sass-loader',
-            options: {
-              sassOptions: {
-                includePaths: ['node_modules'],
-              },
-            },
-          },
-        ],
-      },
-    ],
-  },
-  output: {
-    filename: argv.mode === 'production'
-      ? '[name].js'
-      : '[name].development.js',
-    libraryTarget: 'umd',
-    path: path.join(__dirname, 'dist'),
-    publicPath: '/dist/',
-  },
-  performance: {
-    hints: 'error',
-    maxAssetSize: argv.mode === 'production'
-      ? MAX_PRODUCTION_OUTPUT_SIZE
-      : MAX_DEVELOPMENT_OUTPUT_SIZE,
-    maxEntrypointSize: argv.mode === 'production'
-      ? MAX_PRODUCTION_OUTPUT_SIZE
-      : MAX_DEVELOPMENT_OUTPUT_SIZE,
-  },
-  plugins: [
-    new StyleLintPlugin({
-      configFile: 'stylelint.config.js',
-      files: '**/*.scss',
-    }),
-    new VisualizerPlugin({
```

Comment:
Why were those plugins removed? Both are useful and used.

Reply:
it is not compatible with Webpack 5 and I could not find a suitable replacement

Reply:
We use `webpack-visualizer-plugin2` in other project. Same API, different package.

Reply:
i added it